src/components/Home.js

```diff
@@ -1,31 +1,80 @@
-import React, { useState } from 'react';
+import React, { useState, useEffect } from 'react';
 import SearchBar from './SearchBar';
 import DropDown from './DropDown';
 
 const Home = () => {
 
-    const [query, setQuery] = useState('');
+    let url = 'https://api.edamam.com/api/recipes/v2?type=public&app_id=b3790de0&app_key=01c4c2311660639bc36c94acfc136141';
```

Comment:
I would highly recommend storing the app_key somewhere else (environment variable). We never want to declare things like this explicitly in code. Just a good security practice

Reply:
I have implemented the changes in my AWS console and also locally.